k8s.gcr.io/Vanity-Domain-Flip.md

```diff
@@ -0,0 +1,191 @@
+- [`pull-cip-auditor-e2e`](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml)
+  Like `pull-cip-e2e`, but runs E2E tests for the auditing
+  mechanism built into the promoter; this mechanism (known as "cip-auditor"),
+  runs as a Cloud Run service. It uses the
+  `k8s-infra-gcr-promoter@k8s-gcr-audit-test-prod.iam.gserviceaccount.com` GCP
+  project resources for creating/deleting Cloud Run services, as well as
+  clearing Pub/Sub messages and Stackdriver logs to run its tests. Note that
+  it uses a separate GCP project than the `pull-cip-e2e`, so that the two
+  tests are isolated from each other.
+- [`pull-cip-unit-tests`](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml)
+  This runs unit tests for the promoter codebase, and are part of
+  the PR presubmit checks.
+- [`pull-cip-lint`](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml)
+  This runs [golangci-lint][golangci-lint] for the promoter
+  codebase (which is primarily written in Go).
+- [`pull-k8sio-backup`](https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml)
+  Checks that changes to the [backup scripts][k8sio-backup] are
+  valid. Like the `pull-cip-e2e` and `pull-cip-auditor-e2e` jobs, this job
+  uses GCP resources to check that the backup scripts work as intended in
+  `ci-k8sio-backup`.
+
+## Critical User Journey for Promotion
+
+In order for a user to push to `k8s-artifacts-prod`, they must:
+
+1. Ensure that they have a [subproject staging repo][staging-subproject] (e.g.,
+   `gcr.io/k8s-staging-foo` for the `foo` subproject).
+2. Add the promotion metadata in the [manifests subdirectory][] in the k8s.io repo.
+
+### Security Restrictions
+
+- **Write-once**: Images promoted to production will NOT be deleted, unless under extreme,
+  emergency circumstances that require human supervision (see "Breakglass"
+  section below).
+- **Immutable tags**: New images added to the promoter manifests cannot use an
+  existing tag for the same image. In other words, tags (once created for an
+  image) cannot be deleted.
+- **Mandatory subproject prefix**: Images must be prefixed in production by the
+  name of the subproject. For example, the subproject named `foo` must only push
+  images to `{asia,eu,us}.gcr.io/k8s-artifacts-prod/foo/...`.
+
+# Breakglass
+
+Images in `k8s-artifacts-prod` are not normally deletable. For emergencies,
+however, you can reach the GCR admins listed in the
+`k8s-infra-artifact-admins@kubernetes.io` group [here][groups] who have write
+access to the GCS buckets behind GCR.
+
+# Backups
+
+The GCR images in `k8s-artifacts-prod` are backed up every hour. This is done
+with the `ci-k8sio-backup` job. All images are backed up, even legacy images
```

Comment:
"job" is super ambiguous.  Prow job?  Link to config

Reply:
Ack